circle.yml

```diff
@@ -17,6 +17,6 @@ test:
     - docs/dist/pre-commit.sh . || echo "Ignore codesniffer errors"
     # Errors from docs/dist/pre-push.sh are currently ignored
     # because merge commits from github aren't signed
-    - docs/dist/pre-push.sh || echo "Ignore non-signed commits"
+    #- docs/dist/pre-push.sh || echo "Ignore non-signed commits"
```

Comment:
Yes, you are right.

Personally, I am very fine to loosen our requirements about formatting commits in general here and therefore allow also unsigned commits in general (this is also an unsigned commit over GitHub online interface). I can do the deleting of the lines here, when you agree.